cartographer_grpc/mapping/pose_graph_stub.cc

```diff
@@ -44,15 +48,14 @@ cartographer::mapping::MapById<
     cartographer::mapping::SubmapId,
     cartographer::mapping::PoseGraphInterface::SubmapPose>
 PoseGraphStub::GetAllSubmapPoses() {
-  grpc::ClientContext client_context;
   google::protobuf::Empty request;
-  proto::GetAllSubmapPosesResponse response;
-  stub_->GetAllSubmapPoses(&client_context, request, &response);
+  framework::Client<handlers::GetAllSubmapPosesHandler> client(client_channel_);
+  CHECK(client.Write(request));
```

Comment:
(this is blocking for non-streaming handlers, right?)

Reply:
Correct.